README.md

```diff
@@ -47,7 +47,7 @@ Please check out the [issue queue](https://github.com/EddieHubCommunity/support/
       <a href='https://github.com/EddieHubCommunity'>here</a>.
 
 ## FAQ
-- **What is a GitHub Organization?** Essentially it's a way to group many repositories in an entity called an organization. An organization also allows us to group people who contribute and are a part of the community. Some examples would be: [Netflix](https://github.com/Netflix), [Facebook](https://github.com/facebook) and of course [EddieHub](https://github.com/EddieHubCommunity).
+- **What is a GitHub Organization?** Essentially it's a way to group many repositories in an entity called an organization. An organization also allows us to group people who contribute and are a part of the community. Some examples would be - [Netflix](https://github.com/Netflix), [Facebook](https://github.com/facebook), and of course [EddieHub](https://github.com/EddieHubCommunity).
```

Comment:
I think using `:` here is better. 😅

Reply:
- Sentences 

![Capture](https://user-images.githubusercontent.com/69510006/150665219-00401532-f380-47e1-a5ba-ba1451ee3b70.JPG)

 - Explanation
 ![Capture](https://user-images.githubusercontent.com/69510006/150664915-4a3a198a-b0cb-4a88-bafe-36d3866db54d.JPG)

As per grammatical rule, 3rd sentence is correct but I have used 2nd one in correction.